lms/templates/courseware/progress.html

```diff
@@ -94,6 +95,49 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
         <div class="grade-detail-graph" id="grade-detail-graph" aria-hidden="true"></div>
       %endif
 
+      %if credit_course is not None:
+        <div class="credit-eligibility">
+            <div class="credit-eligibility-container">
+                <div class="eligibility-heading">
+                    <h2>Eligibility Requirements for credit</h2>
+                </div>
+                %if credit_course['eligibility_status'] == 'Not Eligible':
+                    <span class="eligibility_msg">${student.username}, You are no longer eligible for this course</span>
+                %elif credit_course['eligibility_status'] == 'Eligible':
+                    <span class="eligibility_msg">username, You have met the requirements for the credit. Please
+                        <a href="#">Check your dashboard</a> for purchase.
+                    </span>
+                %elif credit_course['eligibility_status'] == 'Partial Eligible':
+                    <span>${_("You have not yet met the requirements for credit")}</span>
+                %endif
+                <button class="credit-help"><i class="fa fa-question"></i></button><br>
+                <div class="requirement-container is-hidden">
+                    %for requirement in credit_course['requirements']:
+                        <div class="requirement">
+                            <div class="requirement-name">${_(requirement['name'])}</div>
+                            <div class="requirement-status">
+                                %if requirement['status']:
+                                    %if requirement['status'] == 'submitted':
+                                        <span class="requirement-submitted">${_("Verification Submitted")}</span>
+                                    %elif requirement['status'] == 'failed':
+                                        <img src="${static.url('images/incorrect-icon.png')}">
```

Comment:
We can save a small bit of bandwidth by using [FontAwesome](http://fontawesome.io/3.2.1/) here instead of images. The package should be installed. [icon-ok](http://fontawesome.io/3.2.1/icon/ok/) and icon-remove (http://fontawesome.io/3.2.1/icon/remove/) should meet your needs and you can use CSS to set the correct color.